rm-ethereum.conf

```diff
@@ -9,6 +9,6 @@
 "appendingZerosForETH":"1000000000000000000",
 "appendingZerosForTKN":"1000000000000000000",
-"maxDepositBits":"192",
```

Comment:
The maxDepositBits is the number of bits when we convert maximum deposit into binary. If you change the value from 192 to 10, it means a person can put 1024 Wei of token at maximum. So, we do not need to change this value, since 2 to the power 192 is defined in payment channel contract.

Reply:
Got it.
Do we want to add our Maximum deposit limit.